pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -17,71 +17,18 @@ limitations under the License.
 package v1beta1
 
 import (
-	"encoding/json"
-	"fmt"
-
-	"github.com/hashicorp/go-multierror"
 	resource "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/result"
 	v1 "k8s.io/api/core/v1"
 )
 
-// RunResult is used to write key/value pairs to TaskRun pod termination messages.
-// The key/value pairs may come from the entrypoint binary, or represent a TaskRunResult.
-// If they represent a TaskRunResult, the key is the name of the result and the value is the
-// JSON-serialized value of the result.
-type RunResult struct {
```

Comment:
Also need an alias for RunResult for completeness. (ideally you should be able to make this change without updating any of its usage if you wanted).

Reply:
Makes sense, thanks. I just restored that and aliased.